covid19-notebooks/covid19-notebook-etl-requirements.txt

```diff
@@ -4,3 +4,7 @@
 google-cloud-bigquery[bqstorage,pandas]>=2.13.1,<3.0.0
 jupyter>=1.0.0,<2.0.0
 seaborn>=0.11.1,<1.0.0
+matplotlib==3.4.3
+plotly==5.5.0
+scipy==1.7.2
+awscli==1.18.*
```

Comment:
the snyk check is failing because of vulns in this version of awscli. I think we would be fine updating it, all we use it for is an `aws s3 cp` command. The latest version is 1.22.51, we can try `awscli>=1.22.0,<2.0.0` instead of 1.18

Reply:
I changed the version of `awscli` in `covid19-notebook-etl-environment.yml` file, there is no problem during docker building. Everything runs fine based on the new image.